src/app/components/file-attachments/single-file-attachment/index.html

```diff
@@ -77,7 +77,8 @@ <h3>States</h3>
       <sky-demo-page-component-state stateName="Base" imagePath="~/assets/img/guidelines/single-file-attach/state-base.png"></sky-demo-page-component-state>
       <sky-demo-page-component-state stateName="Attached image" imagePath="~/assets/img/guidelines/single-file-attach/state-attached-image.png"></sky-demo-page-component-state>
       <sky-demo-page-component-state stateName="Attached non-image" imagePath="~/assets/img/guidelines/single-file-attach/state-attached-non-image.png"></sky-demo-page-component-state>
-      <sky-demo-page-component-state stateName="Error" imagePath="~/assets/img/guidelines/single-file-attach/state-error.png"></sky-demo-page-component-state>
+      <sky-demo-page-component-state stateName="Missing required file error" imagePath="~/assets/img/guidelines/single-file-attach/state-error-missing.png"></sky-demo-page-component-state>
```

Comment:
This is the only image in this table where the label appears above the input. Should we crop it out of the image for consistency?

Reply:
Gotcha. Makes sense. Thanks!